docker-compose.yml

```diff
@@ -31,6 +34,25 @@ services:
       - runserver
       - --debug
 
+  celery:
+    build:
+      context: .
+    image: celery:local
+    user: root
+    depends_on:
+      - db
+      - redis
+    env_file:
+      - .env
+    volumes:
+      - .:/anem-per-feina/
+    entrypoint:
+      - invoke
+      - wait-for
+      - redis:6379
+      - celery-queues
+      - --debug
```

Comment:
this debug is not right. You should remove it

Reply:
Done!